src/GitLink/LinkOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace GitLink
```

Comment:
No need for StyleCop rules. And yes, let's keep modernizing for a later iteration ;-)